docker/prognostic_run/Dockerfile

```diff
@@ -93,20 +95,14 @@ RUN apt-get update && apt-get install -y  apt-transport-https ca-certificates gn
 # Equivalent to passing --no-cache-dir to every pip install
 ENV PIP_NO_CACHE_DIR=off
 
+# do this early on the build since it is upstream of fv3net
```

Comment:
```suggestion
# do this early on the build since it is upstream of fv3net
# import as minimal test nothing went wrong with the build/install
```
If this is why you are running the import I think it's worth documenting it, could be hard to tell otherwise.

Reply:
I added a comment like this.